locales/de/index.yml

```diff
@@ -135,198 +135,315 @@ global:
     pec: "PEC"
     phone: "Telefon"
 tablet:
-  message: Die Oberfläche der IO-App ist für Smartphones optimiert, aber alle Funktionen, die mit Ihrem Gerät kompatibel sind, sind weiterhin verfügbar.
+  message: "Die Benutzeroberfläche von IO ist für Smartphones optimiert, aber alle mit deinem Gerät kompatiblen Funktionen sind weiterhin verfügbar."
 rooted:
-  title: Es sieht so aus, als ob die Sicherheit Ihres Geräts kompromittiert worden ist!
+  title: "Es sieht so aus, als ob die Sicherheit deines Geräts kompromittiert ist!"
   bodyiOS: !include jailbroken_body.md
   bodyAndroid: !include rooted_body.md
   continueAlert:
-    title: Weiter
-    body: Sind Sie sicher, dass Sie die Risiken kennen und weitermachen wollen?
-    confirmText: Ja
-    cancelText: Abbrechen
+    title: "Weiter"
+    body: "Hast du die Risiken verstanden und möchtest fortfahren?"
+    confirmText: "Ja"
+    cancelText: "Abbrechen"
   cancelAlert:
-    title: Beenden
-    body: Sie können die App schließen
-    confirmText: Beenden
+    title: "Beenden"
+    body: "Du kannst die App schließen"
```

Comment:
@dhinterlechner are you going to change all the strings from 3rd person to 2nd? If so, better to merge all the PRs in bulk, so that we'll have a more consistent tone of voice within the app

Reply:
Yes I am rewriting the strings  to the 2nd person ("du") - OK, will proceed 👍